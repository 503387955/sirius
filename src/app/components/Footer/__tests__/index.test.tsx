import React from 'react';
import { render } from '@testing-library/react';

import { Footer } from '..';

describe('<Footer  />', () => {
  it('should match snapshot', () => {
    const left = [<span key="1">left</span>];
    const right = [<span key="1">right</span>];
    const bottom = <span key="1">@2020 Conflux. All Rights Reserved</span>;
    const loadingIndicator = render(
      <Footer left={left} right={right} bottom={bottom} />,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
