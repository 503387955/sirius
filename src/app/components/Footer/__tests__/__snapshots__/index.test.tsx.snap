// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer  /> should match snapshot 1`] = `
.c0 {
  border: 1px solid green;
}

<footer
  class="c0"
>
  <div
    class=""
  >
    <span>
      left
    </span>
  </div>
  <div
    class=""
  >
    <span>
      right
    </span>
  </div>
  <div
    class=""
  >
    <span>
      @2020 Conflux. All Rights Reserved
    </span>
  </div>
</footer>
`;
